Tangem/Common/UI/SendGroupedNumberTextField/GroupedNumberFormatter.swift

```diff
@@ -31,6 +31,14 @@ struct GroupedNumberFormatter {
         numberFormatter.maximumFractionDigits = maximumFractionDigits
     }
 
+    func format(_ decimal: Decimal) -> String {
+        guard let string = numberFormatter.string(for: decimal) else {
+            return ""
```

Comment:
может тут 
```suggestion
            return decimal.description
```
навсякий

Reply:
ну окей )